net/ulogd/files/ulogd.init

```diff
@@ -1,7 +1,7 @@
 #!/bin/sh /etc/rc.common
 # Copyright (C) 2014 OpenWrt.org
 
-START=20
+START=12
```

Comment:
ulogd may require to be started even much later as it may require a database server to be up and running... anyhow, it's really a different type of beast than rsyslog or syslog-ng.

Reply:
Okay, dropping ulogd from changeset.

Reply:
I'm made the requested changes.  Can we please proceed?